packages/connectivity/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Tue Jul 09 10:56:47 IST 2019
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.1.1-all.zip
```

Comment:
Gradle upgrades like this introduce breaking changes to Gradle. Ideally we want to be using the latest version of Gradle, but changes like this are risky on their own. Can you revert this for now since it's unrelated to the main bug fix in the PR?

Reply:
done